client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -66,6 +66,26 @@ public class SparkShuffleManager implements ShuffleManager {
   private ExecutorShuffleIdTracker shuffleIdTracker = new ExecutorShuffleIdTracker();
 
   public SparkShuffleManager(SparkConf conf, boolean isDriver) {
+    int maxStageAttempts =
+        conf.getInt(
+            "spark.stage.maxConsecutiveAttempts",
```

Comment:
"spark.stage.maxConsecutiveAttempts" becomes a variable of config.package through this PR (https://github.com/apache/spark/pull/42061), so all versions of spark2 and some versions of spark3 cannot access this config through the variable of config.package